doc/classes/Dictionary.xml

```diff
@@ -340,6 +340,20 @@
 				[b]Note:[/b] [method merge] is [i]not[/i] recursive. Nested dictionaries are considered as keys that can be overwritten or not depending on the value of [param overwrite], but they will never be merged together.
 			</description>
 		</method>
+		<method name="merged" qualifiers="const">
+			<return type="Dictionary" />
+			<param index="0" name="dictionary" type="Dictionary" />
+			<param index="1" name="overwrite" type="bool" default="false" />
+			<description>
+				Creates a new Dictionary, which is a duplicate of this Dictionary merged with the other [param dictionary]. Useful for quickly making Dictionaries with default values.
+				[codeblock]
+				var base = {"fruit": "apple", "vegetable": "potato"}
+				func get_salad(extras):
+				    return extras.merged(base) # "fruit" and "vegetable" keys will be included with the extras Dictionary if not already in there.
```

Comment:
Very long comment, will look horrible in the class reference, especially built-in one.
I propose another example, maybe?
```suggestion
				var base = { "fruit": "apple", "vegetable": "potato" }
				var extra = { "fruit": "orange", "dressing": "vinegar"}

				# Prints { "fruit": "apple", "vegetable": "potato", "dressing": "vinegar" }
				print(extra.merged(base))

				# Prints { "fruit": "orange", "vegetable": "potato", "dressing": "vinegar" }
				print(extra.merged(base, true))
```
Extremely high likelihood I mixed the dictionaries wrong.

Ideally it'd be nice to have an example that uses String keys and numerical values, as they're easier to distinguish, but I have no suggestions

Reply:
The extra lines between prints was to make it more readable, especially in built-in docs with no colors